consoleproxy/consoleproxy/console_proxy_agent.py

```
@@ -417,20 +447,37 @@ def _establish_new_vnc_proxy(self, req):
-        else:
-            command = "ps aux | grep '[z]stack.*websockify_init' | grep '%s' | grep -v '%s' | awk '{ print $2 }'" % (proxy_host_port_grep, cmd.sslCertFile)
-        ret,out,err = bash_roe(command)
-        for pid in out.splitlines():
-            try:
-                os.kill(int(pid), 15)
-            except OSError:
-                continue
+        def kill_websockify_process(host_port_grep):
+            command = "ps aux | grep '%s' | grep '%s' | awk '{ print $2 }'" % (
+                WEBSOCKIFY_PROCESS_PATTERN, host_port_grep)
+            ret,out,err = bash_roe(command)
```

Ruff (0.15.14)

[warning] Unpacked variable `ret` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)